File: code-samples/php/consignment-order@{purchaseOrderNumber}.php
const purchaseOrderNumber = "YOUR_purchaseOrderNumber_PARAMETER";
$query = array(
    "purchaseOrderNumber" => "purchaseOrderNumber=123abc"
);

$curl = curl_init();

$payload = array(
    "isShipped" => true,
    "trackingNumber" => "string",
    "actualDepartureDate" => "2022-02-01T17:51:03.413Z",
    "estimatedArrivalDate" => "2022-02-01T17:51:03.413Z",
    "deliveryAgentPhoneNumber" => "string",
    ""3plName"" => "string"
);

curl_setopt_array($curl, [
    CURLOPT_HTTPHEADER => [
    "Authorization: string",
    "Content-Type: application/json"
    ],
    CURLOPT_POSTFIELDS => json_encode($payload),
    CURLOPT_URL => "https://vendor-api-staging.jumia.com/consignment-order/" . purchaseOrderNumber . "?" . http_build_query($query),
    CURLOPT_RETURNTRANSFER => true,
    CURLOPT_CUSTOMREQUEST => "PATCH",
]);

$response = curl_exec($curl);
$error = curl_error($curl);

curl_close($curl);

if ($error) {
    echo "cURL Error #:" . $error;
} else {
    echo $response;
}

File: code-samples/go/consignment-order@{purchaseOrderNumber}.go
package main

import (
  "fmt"
  "bytes"
  "net/http"
  "io/ioutil"
)

func main() {
  purchaseOrderNumber := "YOUR_purchaseOrderNumber_PARAMETER";
  reqUrl := "https://vendor-api-staging.jumia.com/consignment-order/" + purchaseOrderNumber
  var data = []byte(`{
	"isShipped": true,
	"trackingNumber": "string",
	"actualDepartureDate": "2022-02-01T17:51:03.413Z",
	"estimatedArrivalDate": "2022-02-01T17:51:03.413Z",
	"deliveryAgentPhoneNumber": "string",
	"3plName": "string"
  }`)
  req, _ := http.NewRequest("PATCH", reqUrl, bytes.NewBuffer(data))

  query := req.URL.Query()
  query.Add("purchaseOrderNumber", "purchaseOrderNumber=123abc")
  req.URL.RawQuery = query.Encode()

  req.Header.Add("Content-Type", "application/json")
  req.Header.Add("Authorization", "string")
  res, _ := http.DefaultClient.Do(req)
  defer res.Body.Close()
  body, _ := ioutil.ReadAll(res.Body)

  fmt.Println(res)
  fmt.Println(string(body))
}